### PR TITLE
Change Key Vault Secret Readme

### PR DESCRIPTION
**Pull request by Luyunmt**
Change Key Vault Secret Readme by [issue](https://github.com/Azure/azure-sdk-for-java/issues/10054)

### CODE REVIEW

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Azure Key Vault Secret client library for Java
 Azure Key Vault is a cloud service that provides a secure storage of secrets, such as passwords and database connection strings.
 
-Secret client library allows you to securely store and tightly control the access to tokens, passwords, API keys, and other secrets. This library offers operations to create, retrieve, update, delete, purge, backup, restore and list the secrets and its versions.
+Azure Key Vault Secrets client library allows you to securely store and tightly control the access to tokens, passwords, API keys, and other secrets. This library offers operations to create, retrieve, update, delete, purge, backup, restore and list the secrets and its versions.
```
Comment (jongio) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:4]: Oxford comma

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 2]
```diff
@@ -60,15 +60,15 @@ Here is [Azure Cloud Shell](https://shell.azure.com/bash) snippet below to
     }
     ```
 
-* Use the above returned credentials information to set **AZURE_CLIENT_ID**(appId), **AZURE_CLIENT_SECRET**(password) and **AZURE_TENANT_ID**(tenant) environment variables. The following example shows a way to do this in Bash:
+* Use the above returned credentials information to set **AZURE_CLIENT_ID** (appId), **AZURE_CLIENT_SECRET** (password) and **AZURE_TENANT_ID** (tenantId) environment variables. The following example shows a way to do this in Bash:
```
Comment (jongio) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:63]: Oxford comma

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
 # Azure Key Vault Secret client library for Java
 Azure Key Vault is a cloud service that provides a secure storage of secrets, such as passwords and database connection strings.
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:2]:
```suggestion
Azure Key Vault is a cloud service that provides secure storage for secrets such as passwords and database connection strings.
```

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 4]
```diff
@@ -255,7 +257,7 @@ secretAsyncClient.beginDeleteSecret("secretName")
 
 ### List Secrets Asynchronously
 
-List the secrets in the key vault by calling `listPropertiesOfSecrets`.
+List the secrets in the Azure Key Vault by calling `listPropertiesOfSecrets`.
 
 ```Java
 // The List Secrets operation returns secrets without their value, so for each secret returned we call `getSecret` to get its // value as well.
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:263]:
```suggestion
// The List Secrets operation returns secrets without their value, so for each secret returned we call `getSecret`
// to get its value as well.
```

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 5]
```diff
@@ -255,7 +257,7 @@ secretAsyncClient.beginDeleteSecret("secretName")
 
 ### List Secrets Asynchronously
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:259]:
```suggestion
```

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 6]
```diff
@@ -242,7 +244,7 @@ secretAsyncClient.getSecret("secretName").subscribe(secret -> {
 
 ### Delete a Secret Asynchronously
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:246]:
```suggestion
```

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 7]
```diff
@@ -229,7 +231,7 @@ secretAsyncClient.getSecret("secretName").subscribe(secret ->
 
 ### Update an existing Secret Asynchronously
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:233]:
```suggestion
```

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 8]
```diff
@@ -219,7 +221,7 @@ secretAsyncClient.setSecret("secret_name", "secret_value").subscribe(secret ->
 
 ### Retrieve a Secret Asynchronously
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:223]:
```suggestion
```

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 9]
```diff
@@ -201,8 +203,8 @@ The following sections provide several code snippets covering some of the most c
 
 ### Create a Secret Asynchronously
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:205]:
```suggestion
```

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 10]
```diff
@@ -179,7 +181,7 @@ deletedSecretPoller.waitForCompletion();
 
 ### List Secrets
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:183]:
```suggestion
```

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 11]
```diff
@@ -163,14 +164,15 @@ System.out.printf("Secret's updated expiry time %s \n", updatedSecretProperties.
 
 ### Delete a Secret
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:166]:
```suggestion
```

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 12]
```diff
@@ -150,7 +151,7 @@ System.out.printf("Secret is returned with name %s and value %s \n", secret.getN
 
 ### Update an existing Secret
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:153]:
```suggestion
```

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 13]
```diff
@@ -141,7 +142,7 @@ System.out.printf("Secret is created with name %s and value %s \n", secret.getNa
 
 ### Retrieve a Secret
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:144]:
```suggestion
```

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 14]
```diff
@@ -122,13 +122,14 @@ The following sections provide several code snippets covering some of the most c
 
 ### Create a Secret
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:124]:
```suggestion
```

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 15]
```diff
@@ -1,16 +1,16 @@
...
-Secret client library allows you to securely store and tightly control the access to tokens, passwords, API keys, and other secrets. This library offers operations to create, retrieve, update, delete, purge, backup, restore and list the secrets and its versions.
+Azure Key Vault Secrets client library allows you to securely store and tightly control the access to tokens, passwords, API keys, and other secrets. This library offers operations to create, retrieve, update, delete, purge, backup, restore, and list the secrets and its versions.
 
-Use the secret client library to create and manage secrets.
+Use the Azure Key Vault Secrets client library to create and manage secrets.
 
 [Source code][source_code] | [API reference documentation][api_documentation] | [Product documentation][azkeyvault_docs] | [Samples][secrets_samples]
 
 ## Getting started
 ### Adding the package to your project
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:12]:
```suggestion
```

Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:12]:
```suggestion
```

**Comment by vcolin7** [comment]
Hi @Luyunmt, I made some slight corrections but noticed there are some additional things I'd like to address in sections I cannot comment on, would you mind giving me access to push to your branch so I can make some changes to this README? Thanks in advance :)

**Comment by Luyunmt** [comment]
@vcolin7

**Comment by vcolin7** [comment]
@g2vinay PTAL when you have some time :)

**Comment by vcolin7** [comment]
@jongio PTAL when you have some time :)

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 16]
```diff
@@ -77,235 +76,231 @@ Here is [Azure Cloud Shell](https://shell.azure.com/bash) snippet below to
...
-better performance compared to the default SSL implementation within the JDK. For more information, including how to 
-reduce the dependency size, refer to the [performance tuning][performance_tuning] section of the wiki.
+All client libraries, by default, use the Tomcat-native Boring SSL library to enable native-level performance for SSL operations. The Boring SSL library is an Uber JAR containing native libraries for Linux / macOS / Windows, and provides better performance compared to the default SSL implementation within the JDK. For more information, including how to reduce the dependency size, refer to the [performance tuning][performance_tuning] section of the wiki.
 
 ## Next steps
-Several KeyVault Java SDK samples are available to you in the SDK's GitHub repository. These samples provide example code for additional scenarios commonly encountered while working with Key Vault:
+Several KeyVault Java SDK samples are available to you in the SDK's GitHub repository. These samples provide example code for additional scenarios commonly encountered while working with Azure Key Vault.
```
Comment (jongio) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:287]: Key Vault two words

Comment (vcolin7) [reply sdk/keyvault/azure-security-keyvault-secrets/README.md:287]: Do we want to replace every occurrence of Key Vault with Azure Key Vault in our READMEs?

**Pull request merged**